huaweicloud/services/acceptance/dli/resource_huaweicloud_dli_permission_test.go

```diff
@@ -0,0 +1,409 @@
+					resource.TestCheckResourceAttr(resourceName, "object", fmt.Sprintf("databases.%s", name)),
+					resource.TestCheckResourceAttr(resourceName, "privileges.0", "SELECT"),
+					resource.TestCheckResourceAttrSet(resourceName, "is_admin"),
+				),
+			},
+			{
+				Config: testAccDliAuthResource_basic(name, acceptance.HW_PROJECT_ID, "CREATE_TABLE"),
+				Check: resource.ComposeTestCheckFunc(
+					rc.CheckResourceExists(),
+					resource.TestCheckResourceAttr(resourceName, "user_name", name),
+					resource.TestCheckResourceAttr(resourceName, "object", fmt.Sprintf("databases.%s", name)),
+					resource.TestCheckResourceAttr(resourceName, "privileges.0", "CREATE_TABLE"),
+					resource.TestCheckResourceAttrSet(resourceName, "is_admin"),
+				),
+			},
+			{
+				ResourceName:      resourceName,
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+
+func testAccDliAuthResource_basic(name string, projectId string, privileges string) string {
+	database := testAccDliDatabase_basic(name)
+	userConfig := testAccDliAuthUserConfig(name, projectId)
+
+	return fmt.Sprintf(`
+%s
+%s
+
+resource "huaweicloud_dli_permission" "test" {
+  user_name  = huaweicloud_identity_user.test.name
+  object     = "databases.${huaweicloud_dli_database.test.name}"
+  privileges = ["%s"]
+}
+`, database, userConfig, privileges)
+}
+
+func TestAccResourceDliAuth_flink(t *testing.T) {
+	var obj auth.GrantDataPermissionOpts
+	resourceName := "huaweicloud_dli_permission.test"
+	name := acceptance.RandomAccResourceName()
+
+	rc := acceptance.InitResourceCheck(
+		resourceName,
+		&obj,
+		getDliAuthResourceFunc,
+	)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:          func() { acceptance.TestAccPreCheck(t) },
+		ProviderFactories: acceptance.TestAccProviderFactories,
+		CheckDestroy:      rc.CheckResourceDestroy(),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDliAuthResource_flink(name, acceptance.HW_PROJECT_ID, acceptance.HW_REGION_NAME, "GET"),
+				Check: resource.ComposeTestCheckFunc(
+					rc.CheckResourceExists(),
+					resource.TestCheckResourceAttr(resourceName, "user_name", name),
+					resource.TestMatchResourceAttr(resourceName, "object", regexp.MustCompile(`jobs\.flink\.\d*`)),
+					resource.TestCheckResourceAttr(resourceName, "privileges.0", "GET"),
+					resource.TestCheckResourceAttrSet(resourceName, "is_admin"),
+				),
+			},
+			{
+				Config: testAccDliAuthResource_flink(name, acceptance.HW_PROJECT_ID, acceptance.HW_REGION_NAME, "START"),
+				Check: resource.ComposeTestCheckFunc(
+					rc.CheckResourceExists(),
+					resource.TestCheckResourceAttr(resourceName, "user_name", name),
+					resource.TestMatchResourceAttr(resourceName, "object", regexp.MustCompile(`jobs\.flink\.\d*`)),
+					resource.TestCheckResourceAttr(resourceName, "privileges.0", "START"),
+					resource.TestCheckResourceAttrSet(resourceName, "is_admin"),
+				),
+			},
+			{
```

Comment:
Only keep the import in basic function, and the other does not need to check the import.

Reply:
You need to take a look at our specification, it is inconsistent with the specification, although it is not an error.